src/main/webapp/tests.html

```diff
@@ -7,16 +7,21 @@
   <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-2.10.0.css">
 </head>
 <body>
+    <select id="region-select"></select>
   <div id="qunit"></div>
-  <div id="qunit-fixture"></div>
+  <div id="qunit-fixture">
+
+  </div>
   <script src="https://code.jquery.com/qunit/qunit-2.10.0.js"></script>
+  <script src="https://unpkg.com/qunit-dom/dist/qunit-dom.js"></script>
   <script src="https://sinonjs.org/releases/sinon-9.0.0.js"></script>
+  <script src="https://apis.google.com/js/api.js"></script>
   <script src="scripts/script.js"></script>
   <script src="scripts/perspective.js"></script>
   <script src="scripts/perspective-api-service.js"></script>
   <script src="scripts/classes/comment-class.js"></script>
   <script src="scripts/tests/perspective-test.js"></script>
-  <script src="https://apis.google.com/js/api.js"></script>
+  <script type="module" src="scripts/tests/popular-test.js"></script>
   <script defer type="module" src="scripts/tests/comment-post-test.js"></script>
```

Comment:
is `defer` necessary?

Reply:
I don't think I need it for mine, @chenzizhao why was it added comment-post-test.js?

Reply:
ooh sorry it works without defer here my bad, please remove it